### ISSUE TITLE
Support for 10.13.4 beta

### ISSUE THREAD

**Issue by oscarbg** [Floris497/mac-pixel-clock-patch-V2#256]
Hi,
just updated few days ago to first 10.13.4 public beta..
it's safe to use patcher for 10.13.2/10.13.3?
thanks..

**Comment by Floris497** [comment]
I think yes :) if it works it's safe

**Comment by buhlaze** [comment]
I have dell xps 9560 with 4k intelhd630. I'm trying to get it past the boot screen (black screen) on 10.13.4. This patch isn't working. Is there a modified version.

**Comment by Floris497** [comment]
Try Lilu and CoreDisplayFixup

**Comment by Floris497** [comment]
Ah, so, does it get past bootscreen without patch?

**Comment by buhlaze** [comment]
No, it was working fine up until this new 10.13.4 with Lilu and CoreDisplayFixup (I have the latest version). I have reverted back to 10.13.3 and it works fine, but no matter how many times i try to boot into 10.13.4 wit just will not go through for this 4k screen Dell XPS 15 laptop  to work, just gets stuck at blackscreen, i can hear sounds in the background if i press a button, but screen just stays black, thats why i was wonderin if there was a new perl script or kext to get around this.

**Comment by diehendrick** [comment]
Hi, Guys. I am using MacMini Late 2012 and LG 27UK600 4K monitor.  Cable is MiniDisplayPort to Displayport. if I am using this script. my mac goes blank.  I had tried many times and search in few sites. but unfortunately, i didn't find what I am looking for.  without this script screen can get 4K but 30hz, I think because the cable. and HiDPI on 1920x1080 but 30hz. and how can I hit 60hz? Does anyone has tried?

Actually, I want to set the resolution on 2650 x 1440, right now it works fine with 60hz but seems a bit blurry.
